test_vm/vendor/cookbooks/dbfit_test/metadata.rb

```diff
@@ -7,7 +7,7 @@
 version          '0.3.0'
 depends          'mysql2_chef_gem'
 depends          'yum-mysql-community'
-depends          'mysql', '~> 8.2'
+depends          'mysql'
```

Comment:
I'm still not sure what's going on here. The docs

https://docs.chef.io/cookbook_versions.html

suggest that it wouldn't use `8.3.1` but it does end up in `Berksfile.lock` if I include the `'~> 8.2'` constraint.